web/src/HUD.tsx

```diff
@@ -56,6 +56,7 @@ type HudState = {
     Resources: Array<Resource>
     Log: string
     LogTimestamps: boolean
+    SailEnabled: boolean
```

Comment:
idk if this is really the right place to put this info? but i can't think of a better one 🤷‍♀️